pronto_mia/lib/ui/views/deployment_plan/edit/deployment_plan_edit_view.dart

```diff
@@ -51,88 +57,124 @@ class DeploymentPlanEditView extends StatelessWidget
               ),
-            ),
-            DateTimePicker(
-              type: DateTimePickerType.dateTime,
-              controller: availableFromController,
-              firstDate: deploymentPlan != null
-                  ? deploymentPlan.availableFrom
-                  : DateTime.now(),
-              lastDate: DateTime.now().add(const Duration(days: 365)),
-              dateMask: 'dd.MM.yyyy HH:mm',
-              dateLabelText: 'Gültig ab*',
-            ),
-            DateTimePicker(
-              type: DateTimePickerType.dateTime,
-              controller: availableUntilController,
-              firstDate: deploymentPlan != null
-                  ? deploymentPlan.availableUntil
-                  : DateTime.now(),
-              lastDate: DateTime.now().add(const Duration(days: 365)),
-              dateMask: 'dd.MM.yyyy HH:mm',
-              dateLabelText: 'Gültig bis*',
-            ),
-            Row(
-              mainAxisAlignment: MainAxisAlignment.center,
-              crossAxisAlignment: CrossAxisAlignment.baseline,
-              textBaseline: TextBaseline.ideographic,
-              children: [
-                TextButton(
-                  onPressed: () => handlePdfUpload(model),
-                  child: const Text('Datei hochladen'),
-                ),
-                const SizedBox(width: 8.0),
-                Expanded(
-                  child: TextField(
-                    controller: pdfPathController,
-                    readOnly: true,
-                    onTap: model.openPdf,
-                    style: const TextStyle(
-                      color: Colors.blue,
-                      decoration: TextDecoration.underline,
-                    ),
+              const SizedBox(width: 8.0),
+              Expanded(
+                child: TextField(
+                  controller: pdfPathController,
+                  readOnly: true,
+                  onTap: model.openPdf,
+                  decoration: const InputDecoration(labelText: 'Dateiname'),
+                  style: const TextStyle(
+                    color: CustomColors.link,
+                    decoration: TextDecoration.underline,
                   ),
                 ),
-              ],
-            ),
-          ],
-          primaryButton: ButtonSpecification(
-            title: 'Speichern',
-            onTap: model.submitForm,
-            isBusy: model.busy(model.editBusyKey),
+              ),
+            ],
           ),
-          secondaryButton: (() {
-            if (deploymentPlan != null) {
-              return ButtonSpecification(
-                title: 'Löschen',
-                onTap: model.removeDeploymentPlan,
-                isBusy: model.busy(model.removeBusyKey),
-                isDestructive: true,
-              );
-            }
-          })(),
-          validationMessage: model.validationMessage,
+        ],
+        primaryButton: ButtonSpecification(

```

Comment:
Can you add a close button in dialog view?

Reply:
Added.